### ISSUE TITLE
YOLOV10 Outputs Incorrect with Latest Ultralytics Version

### ISSUE THREAD

**Issue by pranavsingh007** [ultralytics/ultralytics#17375]
### Search before asking

- [X] I have searched the Ultralytics [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar feature requests.

### Description

After updating to the latest version of ultralytics, YOLOV10 is not providing correct outputs as expected. Previous versions of ultralytics worked as expected, but with this update, the results seem inconsistent and do not align with the model's usual performance.

### Use case

We are not able to use v10 and v11 simultaneously so want a feature support so that YOLO V10 can run and give correct outputs

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by Y-T-G** [comment]
You need to provide more information and a minimum reproduceable example.

**Comment by glenn-jocher** [comment]
Agree, v10 validation results are all good in https://github.com/ultralytics/ultralytics/actions/runs/11738363192/job/32700734653
